marlowe-dashboard-client/src/Capability/Marlowe.purs

```diff
@@ -89,24 +89,16 @@ instance manageMarloweAppM :: ManageMarlowe AppM where
         -- create the WalletCompanion and MarloweApp for this wallet
         ajaxCompanionAppId <- Contract.activateContract WalletCompanion walletId
         ajaxMarloweAppId <- Contract.activateContract MarloweApp walletId
```

Comment:
Understanding check: Do we drop this because we don't want to event checkt total funds against freshly created wallet?

Reply:
As you mentioned yesterday, for a real wallet (testnet or even mainnet), it wouldn't make sense